svn/common_base.py

```diff
@@ -17,12 +18,24 @@ def external_command(self, cmd, success_code=0, do_combine=False,
         env['LANG'] = svn.config.CONSOLE_ENCODING
         env.update(environment)
 
-        p = subprocess.Popen(
-            cmd,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
-            cwd=wd,
-            env=env)
+        if sys.platform.lower().startswith('win'):
+            startupinfo = subprocess.STARTUPINFO()
+            startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+
+            p = subprocess.Popen(
```

Comment:
While we're here, let's refactor to use `subprocess.check_output()` rather than `subprocess.Popen()` directly.

Reply:
That could go in another pull request as just a code refactor.